src/util.rs

```diff
@@ -155,3 +155,17 @@ impl<'a> Union<'a> {
         self.first.len() + self.second.len() - self.second.intersection(&self.first).count()
     }
 }
+
+/// Get the approximate size of entry
+#[inline]
+pub fn entry_approximate_size(e: &Entry) -> usize {
+    //  message Entry {
+    //      EntryType entry_type = 1;
+    //      uint64 term = 2;
+    //      uint64 index = 3;
+    //      bytes data = 4;
+    //      bytes context = 6;
+    //      bool sync_log = 5;(Deprecated)
+    // }
+    e.data.len() + e.context.len() + 12
```

Comment:
Plz add more comments about the magic number `12` here.

Reply:
Updated

Reply:
In a long run system, `index` and `term` may larger than 16383 (2 bytes), but it is okay to me since it is an approximate size.